build.proj

```diff
@@ -258,6 +258,8 @@
         ContinueOnError="false" 
         Condition="!$(DelaySign) and '@(DelaySignedAssembliesToSign)' != ''"/>
         
+    <Exec Command="$(MSBuildProjectDirectory)\tools\CheckStrongNameSignature.ps1"/>
```

Comment:
@cormacpayne any reason we need the ps1 file, rather than adding the same code in the ValidateStrongNameSignatureTask ?